### ISSUE TITLE
Smart Airport Surveillance System

### ISSUE THREAD

**Issue by harrychen9999** [Grad-CSE2016/Action-Recognition#7]
can you send me one copy of your paper "Smart Airport Surveillance System"?Thank you.  
my email 1097795279@qq.com

**Comment by aelnouby** [comment]
You can find our BSc thesis [here](https://www.researchgate.net/publication/308903971_Smart_Airport_Surveillance_System_Action_Recognition_Unattended_Object_Detection_Tracking) on research gate.

**Comment by harrychen9999** [comment]
Yes, I knew. However, I can not  download the pdf from the researchgate.
can you send me the original file?
Thank you!

**Comment by aelnouby** [comment]
You can find the thesis [here](https://drive.google.com/file/d/0B4mutBv2Nvu3RjVxYUJPU2k3d2M/view?usp=sharing) as well.

**Comment by harrychen9999** [comment]
After reading your BSc thesis paper,I think you did a great job! Can you share your paper's code?

**Comment by harrychen9999** [comment]
I think you did a great job!!!

**Comment by aelnouby** [comment]
Thanks a lot :D
